docs/api-docs/useful-info/api-data-sources-availability.mdx

```diff
@@ -99,6 +107,36 @@ Below are a list of known issuses with our data sources:
 - No container weight
 - Only Dry container types are mapped to our system
 
+## Rail Coverage
+
+Rail data is populated from requests to the rail carriers.  Some carriers provide more data than others.
+
+
+| Event                       | BNSF | Canadian National Railway | Canadian Pacific Railway | Norfolk Southern Railway | Union Pacific Railroad | CSX Transportation |
+|-----------------------------|------|---------------------------|--------------------------|--------------------------|------------------------|---------------------|
+| rail_loaded                 |  ❌  |           ✅                |       ✅                   |       ✅                   |         ✅               |       ❌              |
```

Comment:
Instead of showing the specific events, let's use the same notifications we did above so the labels are consistent to avoid confusion. That will greatly reduce the number of rows in this table but that's okay - we can add them back as we go.

Reply:
When you say "same notifications we did above", which set in particular are you referencing?

My best guess is the ones under this header: https://github.com/Terminal49/API/pull/108/files#diff-2a856bb2e28232da5753a973b7a6048a61613006f7f8a23f76327ddaa58a9d17R85

* Rail Loaded
* Rail Departed
* Rail Arrived
* Rail Unloaded
* Available for Pickup 
* Not Available
* Holds and Fees Updated
* Full Out
* Empty In
* Estimated Rail Arrival